client/ui/qml/Pages/PageQrDecoder.qml

```diff
@@ -120,40 +120,40 @@ PageBase {
                 }
             }
 
-            QZXingFilter
-            {
-                id: zxingFilter
-                orientation: videoOutput.orientation
-                captureRect: {
-                    // setup bindings
-                    videoOutput.contentRect;
-                    videoOutput.sourceRect;
-                    return videoOutput.mapRectToSource(videoOutput.mapNormalizedRectToItem(Qt.rect(
-                        0.15, 0.15, 0.7, 0.7 //0, 0, 1.0, 1.0
-                    )));
-                }
-
-                decoder {
-                    enabledDecoders: QZXing.DecoderFormat_QR_CODE
-
-                    onTagFound: {
-                        QrDecoderLogic.onDetectedQrCode(tag)
-                    }
-
-                    tryHarder: true
-                }
-
-                property int framesDecoded: 0
-                property real timePerFrameDecode: 0
-
-                onDecodingFinished:
-                {
-                    timePerFrameDecode = (decodeTime + framesDecoded * timePerFrameDecode) / (framesDecoded + 1);
-                    framesDecoded++;
-                    if(succeeded)
-                        console.log("frame finished: " + succeeded, decodeTime, timePerFrameDecode, framesDecoded);
-                }
-            }
+//            QZXingFilter // Which module will be used to read QrCodes ?
+//            {
+//                id: zxingFilter
+//                orientation: videoOutput.orientation
+//                captureRect: {
+//                    // setup bindings
+//                    videoOutput.contentRect;
+//                    videoOutput.sourceRect;
+//                    return videoOutput.mapRectToSource(videoOutput.mapNormalizedRectToItem(Qt.rect(
+//                        0.15, 0.15, 0.7, 0.7 //0, 0, 1.0, 1.0
+//                    )));
+//                }
+
+//                decoder {
+//                    enabledDecoders: QZXing.DecoderFormat_QR_CODE
+
+//                    onTagFound: {
+//                        QrDecoderLogic.onDetectedQrCode(tag)
+//                    }
+
+//                    tryHarder: true
+//                }
+
+//                property int framesDecoded: 0
+//                property real timePerFrameDecode: 0
+
+//                onDecodingFinished:
+//                {
+//                    timePerFrameDecode = (decodeTime + framesDecoded * timePerFrameDecode) / (framesDecoded + 1);
+//                    framesDecoded++;
+//                    if(succeeded)
+//                        console.log("frame finished: " + succeeded, decodeTime, timePerFrameDecode, framesDecoded);
+//                }
+//            }
 
```

Comment:
please revert back it - no need to disable this code in this commit
it will be cleaned up in another branch